src/page/Profile/components/FollowingModal.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { VscChromeClose } from "react-icons/vsc";
+import { User } from "../../../components/componentExport";
+
+export const FollowingModal = ({ followingList, setIsFollowingList }) => {
+  return (
+    <div className="modal-container">
+      <div className="modal flex-col">
+        <button
+          className="modal-close-btn"
+          onClick={() => setIsFollowingList(false)}
+        >
+          <VscChromeClose />
+        </button>
+        {followingList.map((user) => (
+          <User suggestedUser={user} key={user._id} />
+        ))}
+      </div>
+    </div>
```

Comment:
Here, you can reuse a single component to show the list of Following and Followers by passing the list as a prop to the modal.

Reply:
Ya I got it , thanks himadri.